Create Bar graph

Naomi-Dennis opened the pull request.

Naomi-Dennis committed:
Change visualization to bar graph

The bar graph is generated with elm-vegalite

Naomi-Dennis committed:
Remove KMS creation for s3 state tf state backup

Naomi-Dennis committed:
Update Travis

damonkelley commented:
@Naomi-Dennis Could you include a screenshot in the description!? 😄

Naomi-Dennis commented:
[Bar Graph Preview](https://drive.google.com/file/d/1RqXJ8YQ9qRIcifgN2MHJyTn0p5aGmVdc/view?usp=sharing)

damonkelley commented:
Thanks! BTW, you can also attach images directly to the PR and it will display in the comment or description :smile:

<img width="569" alt="Mind the Gapp Graph Preview" src="https://user-images.githubusercontent.com/4121849/76235318-53e18780-6201-11ea-8053-64325b9fa254.png">

damonkelley commented:
We haven't discussed any styling yet, but I envisioned that the bars in the graph might be green to match the G train bullet. https://en.wikipedia.org/wiki/File:NYCS-bull-trans-G-Std.svg

damonkelley commented on src/Main.elm:
```diff
@@ -1,56 +1,60 @@
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    , [ "1:00 AM", "2:00 AM", "01:00:00" ]
-    ]
+    """{"day": ["7 Days Before", "6 Days Before", "5 Days Before", "4 Days BeforE", "3 Days Before", "2 Days Before", "1 Day Before", "Today"],"meanArrivalDifferences": [2.34, -0.10, 5.30, 7.20, -5.32, -4.20, 0.00, 0.32]}"""
```
This is a little hard to read. Is it possible to reformat it with line breaks? 
```suggestion
    """{
      "day": [
         "7 Days Before",
         "6 Days Before",
         "5 Days Before",
         "4 Days BeforE",
         "3 Days Before",
         "2 Days Before",
         "1 Day Before",
         "Today"
         ],
       "meanArrivalDifferences": [
          2.34,
          -0.10,
          5.30, 
          7.20, 
          -5.32, 
          -4.20, 
          0.00, 
          0.32
         ]
       }"""
```

Naomi-Dennis committed:
Make stylistic changes

Bar color is set to the same shade of green as the G train logo.
A title is added and the font size is increased on axis labels and
titles.

Lastly, the bar style was changed to rounded instead of square.

Naomi-Dennis commented:
Updated graph screenshot. 

![Screen Shot 2020-03-10 at 1 08 20 PM](https://user-images.githubusercontent.com/17726736/76339436-742c4780-62d0-11ea-9562-11c14450c9e4.png)
